packages/ag-charts-community/src/chart/legend.ts

```diff
@@ -536,16 +536,17 @@ export class Legend extends BaseProperties {
         });
         spriteWidth += spriteAAPadding * 2;
         spriteHeight += spriteAAPadding * 2;
-        return { spriteAAPadding, spriteWidth, spriteHeight, markerWidth };
+        const spritePixelRatio = getWindow().devicePixelRatio;
```

Comment:
This is the key change.

This is the only place where `devicePixelRation` is used. The `spritePixelRatio` value is used a scaling factor when rendering the sprite.